### ISSUE TITLE
Transcode video cause A/V out of sync

### ISSUE THREAD

**Issue by GoogleCodeExporter** [0xfab-ri/universal-media-server#24]
```
What steps will reproduce the problem?
1. Transcode video cause A/V out of sync
2.
3.

What is the expected output? What do you see instead?
Transcode video cause A/V out of sync

What version of the product are you using? On what operating system?
2.1.1 Windows 7 32bit

Please provide any additional information below.
I'm serving to my Toshiba BDX3300 and when Transcoding a video from a my 
laptop, A/V is out of sync. Also, what render-er would be best for this Blu-Ray 
player? Thank you

```

Original issue reported on code.google.com by `matrixe...@gmail.com` on 30 Nov 2012 at 6:18

**Comment by GoogleCodeExporter** [comment]
```
I did some sync fixes in 2.1.2, is it better with that version?
As for the renderer, I don't know, but if you try some and find out we would 
love to know so we can make a proper renderer for you
```

Original comment by `subjunk` on 1 Dec 2012 at 8:43

**Comment by GoogleCodeExporter** [comment]
```
Waited for reply and didn't get one, can be re-opened
```

Original comment by `subjunk` on 9 Feb 2013 at 9:33
- Changed state: **Invalid**
